.github/workflows/release.yml

```diff
@@ -16,18 +16,18 @@ jobs:
         run: git status --porcelain
       - name: bazel test //...
         env:
-          # Bazelisk will download bazel to here
+          # Bazelisk will download bazel to here.
```

Comment:
nit: we don't actually have to download bazelisk!
https://github.com/cgrindel/rules_bazel_integration_test/pull/1#discussion_r750529518

Reply:
Will defer this to another PR.